src/components/transactions/votedDelegates.js

```diff
@@ -17,19 +17,25 @@ class VotedDelegates extends AccountList {
           <div className={styles.label}>
             <div className='voters-value'>
               {this.props.t('Who voted for this delegate')}
-              {` (${this.state.votersSize})`}
+              {` (${this.props.votersSize || 0})`}
             </div>
-            {super.renderSearchFilter('votersFilterQuery', t('Filter voters'))}
           </div>
           <div className={styles.value}>
             {voters && voters
-              .slice(0, this.state.showVotersNumber)}
+                .slice(0, this.state.showVotersNumber)}
           </div>
           {voters.length > this.state.showVotersNumber ?
```

Comment:
The condition doesn't work right. "Show more" should be shown only before the first click on it. Now it keeps showing it.